app/controllers/reviews_controller.rb

```diff
@@ -2,7 +2,7 @@ class ReviewsController < ApplicationController
   before_action :authenticate_customer!, only: :create
 
   def create
-    @review = Review.new(title: params[:title], text: params[:review], book_id: params[:book_id], customer: @current_customer)
+    @review = Review.new(title: params[:title], text: params[:review], book_id: params[:book_id], customer: current_customer)
```

Comment:
Permitted params can be used here. `@review = Review.new(permitted_params)` instead of list of params